.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 120
+exclude = __init__.py
```

Comment:
Should we match pyincore and add the extend-ignore for E203? Just asking or is that not relevant here?

Reply:
Let's discuss this in the next dev meeting and then based on that outcome, we can either update this PR or merge it. Everything looks good otherwise.

Reply:
Looks like some of the extra extend-ignore is dependent on frameworks used and rules that might need to be ignored. Everything passes so I think we're good for now.